### ISSUE TITLE
Search doesn't work very well anymore 

### ISSUE THREAD

**Issue by paulpoco** [erickok/transdroid-search#113]
Search doesn't work very well anymore

**Comment by paulpoco** [comment]
Bump...

A new version please

**Comment by erickok** [comment]
Yes, this is true. The landscape of open trackers is very bleak and it shows in this app. I'm not even sure if it makes sense to keep a search in the app, instead of having users just use their (mostly private) sites via the browser.

The custom search feature also is confusing people; see #112. Perhaps something like #57 could still be supported but I don't see many people hosting their own. :/

**Comment by cregkly** [comment]
Search in the app was working great for me up until a few weeks ago.

I can search, but adding doesn't work
